modules/ROOT/pages/php.adoc

```diff
@@ -198,120 +65,13 @@ Advanced Neo4j users will have full control over the client and what Cypher gets
-| icon:gift[] Package | `composer require "everyman/neo4jphp"`
-| icon:github[] Source | http://github.com/jadell/neo4jphp
-| icon:book[] Docs | http://github.com/jadell/neo4jphp/wiki/Introduction
-|===
-
-==== Neo4j-OGM-PHP
-
-This library provides a basic object-graph-mapping system and is built on top of Doctrine and the Neo4jPHP library.
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://github.com/lphuberdeau[Louis-Philippe Huberdeau^]
-| icon:github[] Source | https://github.com/lphuberdeau/Neo4j-PHP-OGM
-|===
-
-==== Neo4jOGMBundle
-
-This bundle is wraps the Neo4j-PHP-OGM library mentioned above for the Symfony framework.
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://github.com/klaussilveira[Klaus Silveira^]
-| icon:github[] Source | https://github.com/klaussilveira/neo4j-ogm-bundle
-|===
-
-==== Neo4Laravel
-
-A Neo4j Laravel 4.1 OGM Provider based on Neo4j-OGM-PHP.
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://github.com/lrezek[Lukas Rezek^]
-| icon:github[] Source | https://github.com/lrezek/Neo4Laravel
-|===
-
-==== neo4j-pdo
-
-This is a PHP Data Objects driver for the Neo4j graph database, it allows you to connect to a Neo4j server from PHP using the standard PDO API.
-The project currently only implements a “vanilla” PHP implementation, which means you need to use a special constructor to create your PDO connection.
-PHP extensions to register the driver with PDO in HHVM and Zend VM are planned future work.
-
-* http://github.com/jakewins/neo4j-pdo[neo4j-pdo on GitHub^]
-* Author: http://twitter.com/jakewins[Jacob Hansson^]
-
-==== PHP Cypher
-
-A PHP adapter for Neo4j REST API Cypher endpoint.
-
-* https://github.com/endyjasmi/neo4j[PHP Cypher on GitHub^]
-* Author: https://github.com/endyjasmi[Endy Jasmi^]
-////
-
-[#bolt]
-== Bolt
-
-Bolt protocol library over TCP socket.
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://www.linkedin.com/in/michalstefanak/[Michal Stefanak^]
-| icon:github[] Source | https://github.com/stefanak-michal/Bolt
-|===
-
-[#apps-neo4jphp]
-== Applications built with PHP and Neo4j
-
-=== Omnomhub
-
-Like for Github, but for recipes!
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://twitter.com/michellesanver[Michelle Sanver^]
-| icon:github[] Source | https://github.com/Omnomhub/omnomhub
-|===
-
-=== Thesaurus Manager
-
-If you want to built your own thesaurus, this app is made for you
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://github.com/hasantayyar[Hasan Tayyar^]
-| icon:play-circle[] Example | http://hasantayyar.github.io/thesaurus-manager
-| icon:github[] Source | https://github.com/okulbilisim/thesaurus-manager
-|===
-
-=== Graphgen
-
-Graph generation engine based on a Cypher DSL
-
-[cols="1,4"]
-|===
-| icon:user[] Author | https://github.com/ikwattro[Christophe Willemsen^]
-| icon:github[] Source | https://github.com/graphaware/graphgen
-| icon:book[] Docs | http://graphgen.graphaware.com
-|===
-
 [#php-example]
 include::./_includes/sample_project.adoc[]
```

Comment:
what about the example project? does that need to go too?

Reply:
yep, no active movies repos in PHP